go/vt/vtgate/engine/session_primitive.go

```diff
@@ -76,7 +75,7 @@ func (s *SessionPrimitive) TryStreamExecute(ctx context.Context, vcursor VCursor
 
 // GetFields implements the Primitive interface
 func (s *SessionPrimitive) GetFields(ctx context.Context, vcursor VCursor, bindVars map[string]*querypb.BindVariable) (*sqltypes.Result, error) {
-	return nil, vterrors.New(vtrpcpb.Code_INTERNAL, "not supported for this primitive")
+	return nil, vterrors.VT13001("not supported for this primitive")
```

Comment:
Similar here, should we call this `not supported for session primitive`?

Reply:
Fixed via fa32cade5f